requirements.txt

```diff
@@ -1,3 +1,6 @@
 black
```

Comment:
I'm not sure if we need packages like `black` to be installed directly as dependencies. Tox will install it for us into a dedicated virtual env.

However, this whole requirements format will need to be revisited once we move things into a package, so I don't think it matters too much now.

Reply:
Cleaned up `requirements.txt` in e5d9f71